mintapi/api.py

```diff
@@ -130,14 +135,9 @@ def login_and_get_token(self, email, password, ius_session, thx_guid):  # {{{
 
         data = {'username': email, 'password': password}
 
-        # Extract ius_token/thx_guid using browser if not provided manually
-        if not ius_session:
+        # Extract session_cookies using browser if not provided manually.
+        if not session_cookies:
             session_cookies = self.get_session_cookies(**data)
-        else:
-            session_cookies = {
-                'ius_session': ius_session,
-                'thx_guid': thx_guid
-            }
         self.cookies.update(session_cookies)
 
         self.get('https://pf.intuit.com/fp/tags?js=0&org_id=v60nf4oj&session_id=' + self.cookies['ius_session'])
```

Comment:
This isn't valid anymore because that cookie isn't defined.

Reply:
Interesting; I was getting this cookie back in my response. Are you not seeing it?

Reply:
Unfortunately not.  That line consistently raised a KeyError for me.
`KeyError: "name='ius_session', domain=None, path=None"`
I wonder if cookies are stored over subsequent sessions of Chrome webdriver?

Reply:
Can't explain it but I'm not seeing this anymore.  Pretty sure you can ignore this issue.

Reply:
I was also seeing some spurious behavior. I have a suspicion it's due to timing (not waiting for webdriver to finish loading the response and therefore setting the ius_session)... Looking into reproducing this now.